### PR TITLE
Add proxy

### PR DESCRIPTION
**Pull request by mfrister**
See README on how this works.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (mfrister) [inline LICENSE:190]: TODO

File: main.go [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+			return
+		}
+
+		for _, c := range resp.Cookies() {
+			if c.Name == "_oauth2_proxy" {
+				w.Write([]byte(authCompleteNotice + "You can close this window."))
+				authCookieCh <- c
+				break
+			}
+		}
```
Comment (mfrister) [inline main.go]: Return error message if cookie not found

File: main.go [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+	"net/http/httputil"
+	"net/url"
+	"os"
+	"time"
+
+	"github.com/pkg/browser"
+)
+
+func main() {
+	proxyAddr := "127.0.0.1:8124"
```
Comment (mfrister) [inline main.go]: Make dynamic

**Review by mfrister — COMMENTED** [review]
Comments from reviewing this together this with @sbrandtb.

File: LICENSE (hunk 1 above)
Comment (mfrister) [reply LICENSE:190]: So, reading the paragraph above, these are only instructions on how to apply the license to source code, so these fields are rightly not filled in.

**Review by sbrandtb — APPROVED** [review]

**Pull request merged**